test/util/nightwatch-commands/fill.js

```diff
@@ -2,10 +2,15 @@
  * Clears the current value and if |value| is specified,
  *  enters the |value| in.
  */
-exports.command = function fill(selector, value) {
+exports.command = function fill(selector, value, callback) {
   this.clearValue(selector);
   if (typeof value !== 'undefined') {
-    this.setValue(selector, value);
+    // When callback is passed as undefined, setValue assumes selector is the locating strategy
```

Comment:
That nightwatch documentation is confusing:
```
  browser.setValue('input[type=text]', ['nightwatch', browser.Keys.ENTER]);
```
I thought the callback was a 3rd parameter? Weird.

But anyway, if you're not using the callback, can this comment go away?